gossip/metrics.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 var (
-	confirmBlocksMeter    = metrics.NewRegisteredCounter("confirm/blocks", nil)
-	confirmTxnsMeter      = metrics.NewRegisteredCounter("confirm/transactions", nil)
-	confirmTxLatencyMeter = metrics.NewRegisteredHistogram("confirm/txlatency", nil, metrics.NewUniformSample(10000))
+	confirmBlocksMeter = metrics.NewRegisteredCounter("confirm/blocks", nil)
+	confirmTxnsMeter   = metrics.NewRegisteredCounter("confirm/transactions", nil)
+	txTtfMeter         = metrics.NewRegisteredHistogram("tx_ttf", nil, metrics.NewUniformSample(500))
```

Comment:
looks the same with  
txLatencyMeter   = metrics.NewRegisteredHistogram("tx_ttf", reg, metrics.NewUniformSample(500))
in cmd/tx-storm/metrics.go.

Add comment that these are in different namespaces. Otherwise, looks good.